### PR TITLE
Add Code Template and Solution to Codeforces Question

### PR DESCRIPTION
**Pull request by suyashtrivedi99**
I have added my standard code template that I use for Competitive Programming, alongwith a solution to one of the questions which was asked in a Codeforces competition (in CPP)

### CODE REVIEW

File: Codeforces/Round-674-div-3/C-Increase-and-copy.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#include <bits/stdc++.h>
```
Comment (bhatipradeep) [inline Codeforces/Round-674-div-3/C-Increase-and-copy.cpp:1]: Please can you add question link as a comment here

Comment (suyashtrivedi99) [reply Codeforces/Round-674-div-3/C-Increase-and-copy.cpp:1]: Should I add the link in the solution.cpp itself?

Comment (bhatipradeep) [reply Codeforces/Round-674-div-3/C-Increase-and-copy.cpp:1]: yes

Comment (suyashtrivedi99) [reply Codeforces/Round-674-div-3/C-Increase-and-copy.cpp:1]: Okay. Understood.

**Review by suyashtrivedi99 — COMMENTED** [review]
Have added the question link to solution.cpp

**Pull request merged**